src/main/java/com/hankyz/ymcabooking/handler/WebDriverHandler.java

```diff
@@ -149,4 +156,12 @@ public void book(String day, String month, String year) {
 
     }
 
+    //method increment two days
+    private LocalDateTime getDateTime() {
+        //get time zone
+        LocalDateTime today = LocalDateTime.now(ZoneId.of("America/Montreal"));
+        //return two days later
+        return today.plusDays(2);
```

Comment:
are you sure that today + 2 days is the day we want to reserve? if we are on 9 june 11:59pm, will your method return june 11th or june 12th?

Reply:
I tested it out, it returns june 11 at 11:59 pm.

Reply:
is that what we want tho?